proto/rill/admin/v1/api.proto

```diff
@@ -2198,6 +2198,7 @@ message MagicAuthToken {
   rill.runtime.v1.Expression metrics_view_filter = 10;
   repeated string metrics_view_fields = 11;
   string state = 12;
+  string url = 13;
```

Comment:
1. Can we move this field up above `created_on`? Since it's a key field that's always set.
2. Can we also return the token string separately here? I.e. have a separate `string token = 14;` field?